sdk/eventhub/Azure.Messaging.EventHubs/src/Diagnostics/EventHubsEventSource.cs

```diff
@@ -1834,7 +1965,8 @@ public virtual void AmqpManagementLinkCreateComplete(string eventHubName)
         {
             if (IsEnabled())
             {
-                WriteEvent(91, identifier ?? string.Empty, eventHubName ?? string.Empty, partitionId ?? string.Empty, operationId ?? string.Empty, durationSeconds);
+                // TODO: Discuss. Previously eventCount was not used
```

Comment:
@jsquire we need to discuss this before merging

Reply:
wow... that's a straight up mistake on my part.   It should be in the form that you've got here.